### DISCUSSION TITLE
[Sleef] How to use find_package in cmake to find dependencies on sleef libraries

### DISCUSSION THREAD

**Discussion by jdkuangxx** [microsoft/vcpkg#35446] [Q&A]
As usual, I used the following method to add sleef dependencies,
```
find_package(Sleef REQUIRED)
list(APPEND LIB_LIST Sleef)
add_library(${SIMD_VEC_TARGET} STATIC ${ALL_SRCS})
target_link_libraries(${SIMD_VEC_TARGET} PRIVATE ${LIB_LIST})

```
but encountered the following error message:
![xxx](https://github.com/microsoft/vcpkg/assets/20827861/535673bd-d416-47e3-aea6-17766c48f150)

**Comment by Osyotr** [comment]
The port `sleef` does not export cmake targets in latest release. It was added later in https://github.com/shibatch/sleef/pull/412
